<!DOCTYPE html>
<!-- saved from url=(0042)http://127.0.0.1:7959/Report_v1_JSmith.Rmd -->
<html class=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  
  <script type="application/shiny-singletons"></script>
  <script type="application/html-dependencies">json2[2014.02.04];jquery[3.5.1];shiny[1.5.0]</script>
<script src="./Expressed Variant Reporting_files/json2-min.js"></script>
<script src="./Expressed Variant Reporting_files/jquery.min.js"></script>
<link href="./Expressed Variant Reporting_files/shiny.css" rel="stylesheet">
<script src="./Expressed Variant Reporting_files/shiny.min.js"></script>  <script src="./Expressed Variant Reporting_files/rmd_loader.js"></script>
  <link href="./Expressed Variant Reporting_files/rmd_loader.css" rel="stylesheet">
<meta name="viewport" content="width=device-width, initial-scale=1"><link rel="stylesheet" type="text/css" href="./Expressed Variant Reporting_files/darkly.min.css"><script src="./Expressed Variant Reporting_files/bootstrap.min.js"></script><script src="./Expressed Variant Reporting_files/html5shiv.min.js"></script><script src="./Expressed Variant Reporting_files/respond.min.js"></script><script src="./Expressed Variant Reporting_files/jquery-ui.min.js"></script><link rel="stylesheet" type="text/css" href="./Expressed Variant Reporting_files/jquery.tocify.css"><script src="./Expressed Variant Reporting_files/jquery.tocify.js"></script><script src="./Expressed Variant Reporting_files/tabsets.js"></script><script src="./Expressed Variant Reporting_files/empty-anchor.js"></script><link rel="stylesheet" type="text/css" href="./Expressed Variant Reporting_files/pagedtable.css"><script src="./Expressed Variant Reporting_files/pagedtable.js"></script><link rel="stylesheet" type="text/css" href="./Expressed Variant Reporting_files/anchor-sections.css"><script src="./Expressed Variant Reporting_files/anchor-sections.js"></script><script src="./Expressed Variant Reporting_files/kePrint.js"></script><link rel="stylesheet" type="text/css" href="./Expressed Variant Reporting_files/lightable.css"><script src="./Expressed Variant Reporting_files/htmlwidgets.js"></script><link rel="stylesheet" type="text/css" href="./Expressed Variant Reporting_files/datatables-crosstalk.css"><script src="./Expressed Variant Reporting_files/datatables.js"></script><link rel="stylesheet" type="text/css" href="./Expressed Variant Reporting_files/jquery.dataTables.min.css"><link rel="stylesheet" type="text/css" href="./Expressed Variant Reporting_files/jquery.dataTables.extra.css"><script src="./Expressed Variant Reporting_files/jquery.dataTables.min.js"></script><link rel="stylesheet" type="text/css" href="./Expressed Variant Reporting_files/crosstalk.css"><script src="./Expressed Variant Reporting_files/crosstalk.min.js"></script><link rel="stylesheet" type="text/css" href="./Expressed Variant Reporting_files/rmd_perf.css"><script src="./Expressed Variant Reporting_files/rmd_perf.js"></script><meta name="rstudio_origin" content="127.0.0.1:35387"><script src="./Expressed Variant Reporting_files/rsiframe.js"></script>


<meta name="generator" content="pandoc">
<meta http-equiv="X-UA-Compatible" content="IE=EDGE">


<meta name="author" content="Brandon Michael Blobner, Jenny Leopoldina Smith, Ahmad Al Khleifat">


<title>Expressed Variant Reporting</title>


<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css" data-origin="pandoc">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */

</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    for (var j = 0; j < rules.length; j++) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") continue;
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') continue;
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>


<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type="text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>



<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#section-TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#section-TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



<script type="text/javascript" src="./Expressed Variant Reporting_files/MathJax.js"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover, .MJXp-munder {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > *, .MJXp-munder > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style></head>
<body><div id="MathJax_Message" style="display: none;"></div>
  <div>
    <div data-display-if="!output.__reactivedoc__" data-ns-prefix="" style="display: none;">
      <div id="rmd_loader_wrapper">
        <div id="rmd_loader" style="">
          <p>Please wait...</p>
          <img src="./Expressed Variant Reporting_files/rmd_loader.gif">
        </div>
      </div>
    </div>
    <div id="__reactivedoc__" class="shiny-html-output shiny-bound-output">








<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="section-TOC" class="tocify">
<ul id="tocify-header1" class="tocify-header list-group"><li class="tocify-item list-group-item" data-unique="1_About_The_Test:_Variant_Detection_from_RNA-seq" style="cursor: pointer;"><span class="header-section-number">1</span> About The Test: Variant Detection from RNA-seq</li><ul class="tocify-subheader list-group" data-tag="2" style="display: none;"><li class="tocify-item list-group-item" data-unique="11_snpReportR"><span class="header-section-number">1.1</span> snpReportR</li></ul></ul><ul id="tocify-header2" class="tocify-header list-group"><li class="tocify-item list-group-item" data-unique="2_About_the_Dataset"><span class="header-section-number">2</span> About the Dataset</li></ul><ul id="tocify-header3" class="tocify-header list-group"><li class="tocify-item list-group-item" data-unique="3_HumanMine_Annotation_for_Further_dplyr::filtering_and_Results"><span class="header-section-number">3</span> HumanMine Annotation for Further dplyr::filtering and Results</li></ul><ul id="tocify-header4" class="tocify-header list-group"><li class="tocify-item list-group-item" data-unique="4_Results_Table" style="cursor: pointer;"><span class="header-section-number">4</span> Results Table</li></ul><ul id="tocify-header5" class="tocify-header list-group"><li class="tocify-item list-group-item active" data-unique="5_Chromosome_and_Gene_Vizualization_of_Mutations" style="cursor: pointer;"><span class="header-section-number">5</span> Chromosome and Gene Vizualization of Mutations</li></ul><ul id="tocify-header6" class="tocify-header list-group"><li class="tocify-item list-group-item" data-unique="6_Expression_of_Genes_with_Mutations"><span class="header-section-number">6</span> Expression of Genes with Mutations</li></ul><ul id="tocify-header7" class="tocify-header list-group"><li class="tocify-item list-group-item" data-unique="7_What_Does_This_Result_Mean_for_you" style="cursor: pointer;"><span class="header-section-number">7</span> What Does This Result Mean for you?</li></ul><ul id="tocify-header8" class="tocify-header list-group"><li class="tocify-item list-group-item" data-unique="8_More_information" style="cursor: pointer;"><span class="header-section-number">8</span> More information</li></ul><ul id="tocify-header9" class="tocify-header list-group"><li class="tocify-item list-group-item" data-unique="9_Consent_for_Analysis" style="cursor: pointer;"><span class="header-section-number">9</span> Consent for Analysis</li></ul><ul id="tocify-header10" class="tocify-header list-group"><li class="tocify-item list-group-item" data-unique="10_Genome_References_and_Software" style="cursor: pointer;"><span class="header-section-number">10</span> Genome References and Software</li></ul><ul id="tocify-header11" class="tocify-header list-group"><li class="tocify-item list-group-item" data-unique="11_Session_Information" style="cursor: pointer;"><span class="header-section-number">11</span> Session Information</li></ul></div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="fluid-row" id="section-header">



<h1 class="title toc-ignore">Expressed Variant Reporting</h1>
<h4 class="author">Brandon Michael Blobner, Jenny Leopoldina Smith, Ahmad Al Khleifat</h4>
<h4 class="date">1/6/2021</h4>

</div>


<div id="section-about-the-test-variant-detection-from-rna-seq" class="section level1">
<div name="1_About_The_Test:_Variant_Detection_from_RNA-seq" data-unique="1_About_The_Test:_Variant_Detection_from_RNA-seq"></div><h1><span class="header-section-number">1</span> About The Test: Variant Detection from RNA-seq</h1>
<table class=" lightable-paper lightable-hover" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; float: left; margin-right: 10px;">
<caption>
Patient details:
</caption>
<thead>
<tr>
<th style="text-align:left;">
Information
</th>
<th style="text-align:left;">
Value
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Name:
</td>
<td style="text-align:left;">
Jane Doe
</td>
</tr>
<tr>
<td style="text-align:left;">
DOB:
</td>
<td style="text-align:left;">
01/01/1900
</td>
</tr>
<tr>
<td style="text-align:left;">
Sex:
</td>
<td style="text-align:left;">
F
</td>
</tr>
<tr>
<td style="text-align:left;">
Sample Type:
</td>
<td style="text-align:left;">
RNA
</td>
</tr>
</tbody>
</table>
<table class=" lightable-paper lightable-hover" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; float: right; margin-left: 10px;">
<caption>
Test ordered by:
</caption>
<thead>
<tr>
<th style="text-align:left;">
Information
</th>
<th style="text-align:left;">
Value
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Name:
</td>
<td style="text-align:left;">
Dr.X
</td>
</tr>
<tr>
<td style="text-align:left;">
Doctor identification number:
</td>
<td style="text-align:left;">
12345
</td>
</tr>
<tr>
<td style="text-align:left;">
Hospital:
</td>
<td style="text-align:left;">
NCI
</td>
</tr>
</tbody>
</table>
<div id="section-snpreportr" class="section level2">
<div name="11_snpReportR" data-unique="11_snpReportR"></div><h2><span class="header-section-number">1.1</span> snpReportR</h2>
<div class="figure">
<img src="./Expressed Variant Reporting_files/snpReporter_logo.png" alt="">
<p class="caption">Caption for the picture.</p>
</div>
<pre><code>## Hi DRR131561_fastq, This document will help you to understand the more important findings from a gene variant screening. The common definitions of the type of genetic variants (mutations) are described in the figure. While a variant may have been detected, the associations with the variant are not perfectly causal and their complex interactions between biology and the environment.</code></pre>
<p><code>insert link to diagram here</code></p>
<hr>
</div>
</div>
<div id="section-about-the-dataset" class="section level1">
<div name="2_About_the_Dataset" data-unique="2_About_the_Dataset"></div><h1><span class="header-section-number">2</span> About the Dataset</h1>
<div class="figure" style="text-align: center">
<div id="htmlwidget-b0c4927c5513b966b931" style="width:100%;height:auto;" class="datatables html-widget html-widget-static-bound"><div id="DataTables_Table_0_wrapper" class="dataTables_wrapper no-footer"><div class="dataTables_length" id="DataTables_Table_0_length"><label>Show <select name="DataTables_Table_0_length" aria-controls="DataTables_Table_0" class=""><option value="10">10</option><option value="25">25</option><option value="50">50</option><option value="100">100</option></select> entries</label></div><div id="DataTables_Table_0_filter" class="dataTables_filter"><label>Search:<input type="search" class="" placeholder="" aria-controls="DataTables_Table_0"></label></div><table class="display dataTable no-footer" id="DataTables_Table_0" role="grid" aria-describedby="DataTables_Table_0_info">
  <thead>
    <tr role="row"><th class="sorting_disabled" rowspan="1" colspan="1" aria-label=" "> </th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_0" rowspan="1" colspan="1" aria-label="Number: activate to sort column ascending">Number</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_0" rowspan="1" colspan="1" aria-label="Type: activate to sort column ascending">Type</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_0" rowspan="1" colspan="1" aria-label="Description: activate to sort column ascending">Description</th></tr>
  </thead>
<tbody><tr role="row" class="odd"><td>AC</td><td>A</td><td>Integer</td><td>Allele count in genotypes  for each ALT allele  in the same order as listed</td></tr><tr role="row" class="even"><td>AF</td><td>A</td><td>Float</td><td>Allele Frequency  for each ALT allele  in the same order as listed</td></tr><tr role="row" class="odd"><td>AN</td><td>1</td><td>Integer</td><td>Total number of alleles in called genotypes</td></tr><tr role="row" class="even"><td>BaseQRankSum</td><td>1</td><td>Float</td><td>Z-score from Wilcoxon rank sum test of Alt Vs. Ref base qualities</td></tr><tr role="row" class="odd"><td>DP</td><td>1</td><td>Integer</td><td>Approximate read depth; some reads may have been filtered</td></tr><tr role="row" class="even"><td>DS</td><td>0</td><td>Flag</td><td>Were any of the samples downsampled?</td></tr><tr role="row" class="odd"><td>ExcessHet</td><td>1</td><td>Float</td><td>Phred-scaled p-value for exact test of excess heterozygosity</td></tr><tr role="row" class="even"><td>FS</td><td>1</td><td>Float</td><td>Phred-scaled p-value using Fisher's exact test to detect strand bias</td></tr><tr role="row" class="odd"><td>InbreedingCoeff</td><td>1</td><td>Float</td><td>Inbreeding coefficient as estimated from the genotype likelihoods per-sample when compared against the Hardy-Weinberg expectation</td></tr><tr role="row" class="even"><td>MLEAC</td><td>A</td><td>Integer</td><td>Maximum likelihood expectation (MLE) for the allele counts (not necessarily the same as the AC)  for each ALT allele  in the same order as listed</td></tr></tbody></table><div class="dataTables_info" id="DataTables_Table_0_info" role="status" aria-live="polite">Showing 1 to 10 of 37 entries</div><div class="dataTables_paginate paging_simple_numbers" id="DataTables_Table_0_paginate"><a class="paginate_button previous disabled" aria-controls="DataTables_Table_0" data-dt-idx="0" tabindex="-1" id="DataTables_Table_0_previous">Previous</a><span><a class="paginate_button current" aria-controls="DataTables_Table_0" data-dt-idx="1" tabindex="0">1</a><a class="paginate_button " aria-controls="DataTables_Table_0" data-dt-idx="2" tabindex="0">2</a><a class="paginate_button " aria-controls="DataTables_Table_0" data-dt-idx="3" tabindex="0">3</a><a class="paginate_button " aria-controls="DataTables_Table_0" data-dt-idx="4" tabindex="0">4</a></span><a class="paginate_button next" aria-controls="DataTables_Table_0" data-dt-idx="5" tabindex="0" id="DataTables_Table_0_next">Next</a></div></div></div>
<script type="application/json" data-for="htmlwidget-b0c4927c5513b966b931">{"x":{"filter":"none","data":[["AC","AF","AN","BaseQRankSum","DP","DS","ExcessHet","FS","InbreedingCoeff","MLEAC","MLEAF","MQ","MQRankSum","QD","ReadPosRankSum","SOR","ANN","GENE","OM","PM","SAO","RS","RNAEDIT","RPT","Homopolymer","Entropy","SPLICEADJ","COSMIC_ID","TISSUE","TUMOR","FATHMM","SOMATIC","CHASM_PVALUE","CHASM_FDR","VEST_PVALUE","VEST_FDR","MuPIT"],["A","A","1","1","1","0","1","1","1","A","A","1","1","1","1","1",".",".","0","0","1","1",".","1","1","1","1",".",".",".",".",".",".",".",".",".","."],["Integer","Float","Integer","Float","Integer","Flag","Float","Float","Float","Integer","Float","Float","Float","Float","Float","Float","String","String","Flag","Flag","Integer","Integer","String","String","Integer","Float","Integer","String","String","String","String","String","Float","Float","Float","Float","Character"],["Allele count in genotypes  for each ALT allele  in the same order as listed","Allele Frequency  for each ALT allele  in the same order as listed","Total number of alleles in called genotypes","Z-score from Wilcoxon rank sum test of Alt Vs. Ref base qualities","Approximate read depth; some reads may have been filtered","Were any of the samples downsampled?","Phred-scaled p-value for exact test of excess heterozygosity","Phred-scaled p-value using Fisher's exact test to detect strand bias","Inbreeding coefficient as estimated from the genotype likelihoods per-sample when compared against the Hardy-Weinberg expectation","Maximum likelihood expectation (MLE) for the allele counts (not necessarily the same as the AC)  for each ALT allele  in the same order as listed","Maximum likelihood expectation (MLE) for the allele frequency (not necessarily the same as the AF)  for each ALT allele  in the same order as listed","RMS Mapping Quality","Z-score From Wilcoxon rank sum test of Alt vs. Ref read mapping qualities","Variant Confidence/Quality by Depth","Z-score from Wilcoxon rank sum test of Alt vs. Ref read position bias","Symmetric Odds Ratio of 2x2 contingency table to detect strand bias","Functional annotations: 'Allele | Annotation | Annotation_Impact | Gene_Name | Gene_ID | Feature_Type | Feature_ID | Transcript_BioType | Rank | HGVS.c | HGVS.p | cDNA.pos / cDNA.length | CDS.pos / CDS.length | AA.pos / AA.length | Distance | ERRORS / WARNINGS / INFO'","The name of the gene/s in the genomic region of the SNP as annotated by SNPeff","Has OMIM/OMIA","Variant is Precious(Clinical Pubmed Cited)","Variant Allele Origin: 0 - unspecified  1 - Germline  2 - Somatic  3 - Both","dbSNP ID (i.e. rs number)","A known or predicted RNA-editing site","Repeat family from UCSC Genome Browser Repeatmasker Annotations","Variant is located in or near a homopolymer sequence","Entropy for sequence in window of length 7 centered at the variant position","Variant is within specified distance of a reference exon splice boundary","COSMIC mutation id (unique).","The primary tissue/cancer and subtype from which the sample originated.","The histological classification of the sample.","FATHMM (Functional Analysis through Hidden Markov Models). 'Pathogenic':Cancer or damaging  'Neutral':Passanger or Tolerated.","Information on whether the sample was reported to be Confirmed Somatic. 'Confirmed somatic':if the mutation has been confimed to be somatic in the experiment by sequencing both the tumour and a matched normal from the same patient  'Previously Observed':when the mutation has been reported as somatic previously but not in current paper  'variant of unknown origin':when the mutation is known to be somatic but the tumour was sequenced without a matched normal","CHASM cancer driver p-value (missense) generated by CRAVAT","CHASM cancer driver FDR (missense)","VEST pathogenicity p-value (non-silent)","VEST pathogenicity FDR (non-silent)","MuPIT 3D structure variant link"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Number<\/th>\n      <th>Type<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"order":[],"autoWidth":false,"orderClasses":false,"columnDefs":[{"orderable":false,"targets":0}]},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Table 1. Column names and Descriptions
</p>
</div>
<hr>
</div>
<div id="section-humanmine-annotation-for-further-dplyrfiltering-and-results" class="section level1">
<div name="3_HumanMine_Annotation_for_Further_dplyr::filtering_and_Results" data-unique="3_HumanMine_Annotation_for_Further_dplyr::filtering_and_Results"></div><h1><span class="header-section-number">3</span> HumanMine Annotation for Further dplyr::filtering and Results</h1>
<p>-highly expressed in RNAseq - not highly expressed in normal tissues</p>
<p>References: <em>Smith RN, et al.&nbsp;InterMine: a flexible data warehouse system for the integration and analysis of heterogeneous biological data. Bioinformatics. 2012 Dec 1;28(23):3163-5.</em></p>
<hr>
</div>
<div id="section-results-table" class="section level1 tabset">
<div name="4_Results_Table" data-unique="4_Results_Table"></div><h1><span class="header-section-number">4</span> Results Table</h1>
<p>Included here are two tables for different gene types. Coding refers to genes that produce proteins, while non-coding refers to genes which do get utilized to produce proteins.</p>
<p>The top variants were ranked by the following attributes: * larger indels prioritized (need to parse out size) * genes with larger number of SNVs prioritized * pathogenicity, or splice adjacent * high or moderate impact on the structure of the gene</p>
<ul class="nav nav-tabs" role="tablist"><li role="presentation" class="active"><a role="tab" data-toggle="tab" href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#section-coding-genes" aria-controls="section-coding-genes"><span class="header-section-number">4.1</span> Coding Genes</a></li><li role="presentation"><a role="tab" data-toggle="tab" href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#section-non-coding-genes" aria-controls="section-non-coding-genes"><span class="header-section-number">4.2</span> Non-Coding Genes</a></li></ul><div class="tab-content"><div id="section-coding-genes" class="section level2 tab-pane tabbed-pane active" role="tabpanel">

<div class="figure" style="text-align: center">
<div id="htmlwidget-1346be7d87e150a4181a" style="width:100%;height:auto;" class="datatables html-widget html-widget-static-bound"><div id="DataTables_Table_1_wrapper" class="dataTables_wrapper no-footer"><div class="dataTables_length" id="DataTables_Table_1_length"><label>Show <select name="DataTables_Table_1_length" aria-controls="DataTables_Table_1" class=""><option value="10">10</option><option value="25">25</option><option value="50">50</option><option value="100">100</option></select> entries</label></div><div id="DataTables_Table_1_filter" class="dataTables_filter"><label>Search:<input type="search" class="" placeholder="" aria-controls="DataTables_Table_1"></label></div><table class="cell-border stripe dataTable no-footer" id="DataTables_Table_1" role="grid" aria-describedby="DataTables_Table_1_info">
  <thead>
    <tr role="row"><th class="sorting_disabled" rowspan="1" colspan="1" aria-label=" "> </th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="GENE: activate to sort column ascending">GENE</th><th class="dt-right sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="Number_SNVs_per_Gene: activate to sort column ascending">Number_SNVs_per_Gene</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="COSMIC_ID: activate to sort column ascending">COSMIC_ID</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="rsID: activate to sort column ascending">rsID</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="CHROM: activate to sort column ascending">CHROM</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="POS: activate to sort column ascending">POS</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="ID: activate to sort column ascending">ID</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="REF: activate to sort column ascending">REF</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="ALT: activate to sort column ascending">ALT</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="FATHMM: activate to sort column ascending">FATHMM</th><th class="dt-right sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="SPLICEADJ: activate to sort column ascending">SPLICEADJ</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="TISSUE: activate to sort column ascending">TISSUE</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="TUMOR: activate to sort column ascending">TUMOR</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="Annotation_1: activate to sort column ascending">Annotation_1</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="Annotation_Impact_1: activate to sort column ascending">Annotation_Impact_1</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="Feature_Type_1: activate to sort column ascending">Feature_Type_1</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="Transcript_BioType_1: activate to sort column ascending">Transcript_BioType_1</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="coding_DNA_change_1: activate to sort column ascending">coding_DNA_change_1</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_1" rowspan="1" colspan="1" aria-label="protein_change_1: activate to sort column ascending">protein_change_1</th></tr>
  </thead>
<tbody><tr role="row" class="odd"><td>1</td><td>ACTG1</td><td class=" dt-right">6</td><td>COSV59509939</td><td>rs1139405</td><td>chr17</td><td>81510993</td><td></td><td>G</td><td>A</td><td>PATHOGENIC</td><td class=" dt-right"></td><td>haematopoietic_and_lymphoid_tissue</td><td>haematopoietic_neoplasm_--_acute_myeloid_leukaemia</td><td>synonymous_variant</td><td>LOW</td><td>transcript</td><td>Coding</td><td>c.918C&gt;T</td><td>p.Tyr306Tyr</td></tr><tr role="row" class="even"><td>2</td><td>ACTG1</td><td class=" dt-right">6</td><td>COSV59512043</td><td></td><td>chr17</td><td>81511315</td><td></td><td>C</td><td>T</td><td>PATHOGENIC</td><td class=" dt-right"></td><td>skin</td><td>carcinoma_--_squamous_cell_carcinoma</td><td>synonymous_variant</td><td>LOW</td><td>transcript</td><td>Coding</td><td>c.675G&gt;A</td><td>p.Gln225Gln</td></tr><tr role="row" class="odd"><td>3</td><td>P4HB</td><td class=" dt-right">4</td><td>COSV58941310</td><td>rs2070871</td><td>chr17</td><td>81847258</td><td></td><td>G</td><td>A</td><td>PATHOGENIC</td><td class=" dt-right"></td><td>breast</td><td>carcinoma_--_NS</td><td>synonymous_variant</td><td>LOW</td><td>transcript</td><td>Coding</td><td>c.714C&gt;T</td><td>p.Ile238Ile</td></tr><tr role="row" class="even"><td>4</td><td>PTBP1</td><td class=" dt-right">4</td><td>COSV62462019</td><td>rs3180287</td><td>chr19</td><td>804396</td><td></td><td>C</td><td>T</td><td>PATHOGENIC</td><td class=" dt-right"></td><td>large_intestine</td><td>carcinoma_--_adenocarcinoma</td><td>synonymous_variant</td><td>LOW</td><td>transcript</td><td>Coding</td><td>c.393C&gt;T</td><td>p.Ser131Ser</td></tr><tr role="row" class="odd"><td>5</td><td>CAPN2</td><td class=" dt-right">3</td><td>COSV54340521</td><td>rs17597</td><td>chr1</td><td>223745359</td><td></td><td>C</td><td>T</td><td>PATHOGENIC</td><td class=" dt-right"></td><td>large_intestine</td><td>carcinoma_--_adenocarcinoma</td><td>synonymous_variant</td><td>LOW</td><td>transcript</td><td>Coding</td><td>c.480C&gt;T</td><td>p.Thr160Thr</td></tr><tr role="row" class="even"><td>6</td><td>TPM1</td><td class=" dt-right">3</td><td>COSV51261240</td><td>rs1071646</td><td>chr15</td><td>63059641</td><td></td><td>C</td><td>A</td><td>PATHOGENIC</td><td class=" dt-right"></td><td>large_intestine</td><td>carcinoma_--_adenocarcinoma</td><td>synonymous_variant</td><td>LOW</td><td>transcript</td><td>Coding</td><td>c.453C&gt;A</td><td>p.Ala151Ala</td></tr><tr role="row" class="odd"><td>7</td><td>MRPL28</td><td class=" dt-right">3</td><td>COSV51737791</td><td>rs1802752</td><td>chr16</td><td>369154</td><td></td><td>G</td><td>A</td><td>PATHOGENIC</td><td class=" dt-right"></td><td>lung</td><td>carcinoma_--_NS</td><td>synonymous_variant</td><td>LOW</td><td>transcript</td><td>Coding</td><td>c.355C&gt;T</td><td>p.Leu119Leu</td></tr><tr role="row" class="even"><td>8</td><td>LAMA5</td><td class=" dt-right">3</td><td>COSV53360330</td><td>rs624313</td><td>chr20</td><td>62352356</td><td></td><td>G</td><td>A</td><td>PATHOGENIC</td><td class=" dt-right"></td><td>thyroid</td><td>other_--_neoplasm</td><td>synonymous_variant</td><td>LOW</td><td>transcript</td><td>Coding</td><td>c.573C&gt;T</td><td>p.Ser191Ser</td></tr><tr role="row" class="odd"><td>9</td><td>PDIA6</td><td class=" dt-right">3</td><td>COSV55349252</td><td>rs4807</td><td>chr2</td><td>10790777</td><td></td><td>T</td><td>C</td><td>PATHOGENIC</td><td class=" dt-right"></td><td>haematopoietic_and_lymphoid_tissue</td><td>haematopoietic_neoplasm_--_acute_myeloid_leukaemia</td><td>missense_variant</td><td>MODERATE</td><td>transcript</td><td>Coding</td><td>c.797A&gt;G</td><td>p.Lys266Arg</td></tr><tr role="row" class="even"><td>10</td><td>PDIA6</td><td class=" dt-right">3</td><td>COSV55352693</td><td></td><td>chr2</td><td>10802518</td><td></td><td>C</td><td>A</td><td>PATHOGENIC</td><td class=" dt-right"></td><td>thyroid</td><td>other_--_neoplasm</td><td>missense_variant</td><td>MODERATE</td><td>transcript</td><td>Coding</td><td>c.298G&gt;T</td><td>p.Val100Leu</td></tr></tbody></table><div class="dataTables_info" id="DataTables_Table_1_info" role="status" aria-live="polite">Showing 1 to 10 of 10 entries</div><div class="dataTables_paginate paging_simple_numbers" id="DataTables_Table_1_paginate"><a class="paginate_button previous disabled" aria-controls="DataTables_Table_1" data-dt-idx="0" tabindex="-1" id="DataTables_Table_1_previous">Previous</a><span><a class="paginate_button current" aria-controls="DataTables_Table_1" data-dt-idx="1" tabindex="0">1</a></span><a class="paginate_button next disabled" aria-controls="DataTables_Table_1" data-dt-idx="2" tabindex="-1" id="DataTables_Table_1_next">Next</a></div></div></div>
<script type="application/json" data-for="htmlwidget-1346be7d87e150a4181a">{"x":{"filter":"none","data":[["1","2","3","4","5","6","7","8","9","10"],["ACTG1","ACTG1","P4HB","PTBP1","CAPN2","TPM1","MRPL28","LAMA5","PDIA6","PDIA6"],[6,6,4,4,3,3,3,3,3,3],["COSV59509939","COSV59512043","COSV58941310","COSV62462019","COSV54340521","COSV51261240","COSV51737791","COSV53360330","COSV55349252","COSV55352693"],["rs1139405",null,"rs2070871","rs3180287","rs17597","rs1071646","rs1802752","rs624313","rs4807",null],["chr17","chr17","chr17","chr19","chr1","chr15","chr16","chr20","chr2","chr2"],["81510993","81511315","81847258","804396","223745359","63059641","369154","62352356","10790777","10802518"],[null,null,null,null,null,null,null,null,null,null],["G","C","G","C","C","C","G","G","T","C"],["A","T","A","T","T","A","A","A","C","A"],["PATHOGENIC","PATHOGENIC","PATHOGENIC","PATHOGENIC","PATHOGENIC","PATHOGENIC","PATHOGENIC","PATHOGENIC","PATHOGENIC","PATHOGENIC"],[null,null,null,null,null,null,null,null,null,null],["haematopoietic_and_lymphoid_tissue","skin","breast","large_intestine","large_intestine","large_intestine","lung","thyroid","haematopoietic_and_lymphoid_tissue","thyroid"],["haematopoietic_neoplasm_--_acute_myeloid_leukaemia","carcinoma_--_squamous_cell_carcinoma","carcinoma_--_NS","carcinoma_--_adenocarcinoma","carcinoma_--_adenocarcinoma","carcinoma_--_adenocarcinoma","carcinoma_--_NS","other_--_neoplasm","haematopoietic_neoplasm_--_acute_myeloid_leukaemia","other_--_neoplasm"],["synonymous_variant","synonymous_variant","synonymous_variant","synonymous_variant","synonymous_variant","synonymous_variant","synonymous_variant","synonymous_variant","missense_variant","missense_variant"],["LOW","LOW","LOW","LOW","LOW","LOW","LOW","LOW","MODERATE","MODERATE"],["transcript","transcript","transcript","transcript","transcript","transcript","transcript","transcript","transcript","transcript"],["Coding","Coding","Coding","Coding","Coding","Coding","Coding","Coding","Coding","Coding"],["c.918C&gt;T","c.675G&gt;A","c.714C&gt;T","c.393C&gt;T","c.480C&gt;T","c.453C&gt;A","c.355C&gt;T","c.573C&gt;T","c.797A&gt;G","c.298G&gt;T"],["p.Tyr306Tyr","p.Gln225Gln","p.Ile238Ile","p.Ser131Ser","p.Thr160Thr","p.Ala151Ala","p.Leu119Leu","p.Ser191Ser","p.Lys266Arg","p.Val100Leu"]],"container":"<table class=\"cell-border stripe\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>GENE<\/th>\n      <th>Number_SNVs_per_Gene<\/th>\n      <th>COSMIC_ID<\/th>\n      <th>rsID<\/th>\n      <th>CHROM<\/th>\n      <th>POS<\/th>\n      <th>ID<\/th>\n      <th>REF<\/th>\n      <th>ALT<\/th>\n      <th>FATHMM<\/th>\n      <th>SPLICEADJ<\/th>\n      <th>TISSUE<\/th>\n      <th>TUMOR<\/th>\n      <th>Annotation_1<\/th>\n      <th>Annotation_Impact_1<\/th>\n      <th>Feature_Type_1<\/th>\n      <th>Transcript_BioType_1<\/th>\n      <th>coding_DNA_change_1<\/th>\n      <th>protein_change_1<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,11]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Table 2. Protein Coding Variants, possibly pathogenic
</p>
</div>
</div><div id="section-non-coding-genes" class="section level2 tab-pane tabbed-pane" role="tabpanel">

<div class="figure" style="text-align: center">
<div id="htmlwidget-ed8b198fd5a24386d38b" style="width:100%;height:auto;" class="datatables html-widget html-widget-static-bound"><div id="DataTables_Table_2_wrapper" class="dataTables_wrapper no-footer"><div class="dataTables_length" id="DataTables_Table_2_length"><label>Show <select name="DataTables_Table_2_length" aria-controls="DataTables_Table_2" class=""><option value="10">10</option><option value="25">25</option><option value="50">50</option><option value="100">100</option></select> entries</label></div><div id="DataTables_Table_2_filter" class="dataTables_filter"><label>Search:<input type="search" class="" placeholder="" aria-controls="DataTables_Table_2"></label></div><table class="cell-border stripe dataTable no-footer" id="DataTables_Table_2" role="grid" aria-describedby="DataTables_Table_2_info">
  <thead>
    <tr role="row"><th class="sorting_disabled" rowspan="1" colspan="1" aria-label=" "> </th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="GENE: activate to sort column ascending">GENE</th><th class="dt-right sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="Number_SNVs_per_Gene: activate to sort column ascending">Number_SNVs_per_Gene</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="COSMIC_ID: activate to sort column ascending">COSMIC_ID</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="rsID: activate to sort column ascending">rsID</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="CHROM: activate to sort column ascending">CHROM</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="POS: activate to sort column ascending">POS</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="ID: activate to sort column ascending">ID</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="REF: activate to sort column ascending">REF</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="ALT: activate to sort column ascending">ALT</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="FATHMM: activate to sort column ascending">FATHMM</th><th class="dt-right sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="SPLICEADJ: activate to sort column ascending">SPLICEADJ</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="TISSUE: activate to sort column ascending">TISSUE</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="TUMOR: activate to sort column ascending">TUMOR</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="Annotation_1: activate to sort column ascending">Annotation_1</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="Annotation_Impact_1: activate to sort column ascending">Annotation_Impact_1</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="Feature_Type_1: activate to sort column ascending">Feature_Type_1</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="Transcript_BioType_1: activate to sort column ascending">Transcript_BioType_1</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="coding_DNA_change_1: activate to sort column ascending">coding_DNA_change_1</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_2" rowspan="1" colspan="1" aria-label="protein_change_1: activate to sort column ascending">protein_change_1</th></tr>
  </thead>
<tbody><tr role="row" class="odd"><td>1</td><td>GLYCTK</td><td class=" dt-right">1</td><td>COSV59793331</td><td>rs2276834</td><td>chr3</td><td>52291743</td><td></td><td>A</td><td>G</td><td>NEUTRAL</td><td class=" dt-right"></td><td>urinary_tract</td><td>carcinoma_--_NS</td><td>splice_acceptor_variant&amp;intron_variant</td><td>HIGH</td><td>transcript</td><td>Noncoding</td><td>n.626-2A&gt;G</td><td></td></tr></tbody></table><div class="dataTables_info" id="DataTables_Table_2_info" role="status" aria-live="polite">Showing 1 to 1 of 1 entries</div><div class="dataTables_paginate paging_simple_numbers" id="DataTables_Table_2_paginate"><a class="paginate_button previous disabled" aria-controls="DataTables_Table_2" data-dt-idx="0" tabindex="-1" id="DataTables_Table_2_previous">Previous</a><span><a class="paginate_button current" aria-controls="DataTables_Table_2" data-dt-idx="1" tabindex="0">1</a></span><a class="paginate_button next disabled" aria-controls="DataTables_Table_2" data-dt-idx="2" tabindex="-1" id="DataTables_Table_2_next">Next</a></div></div></div>
<script type="application/json" data-for="htmlwidget-ed8b198fd5a24386d38b">{"x":{"filter":"none","data":[["1"],["GLYCTK"],[1],["COSV59793331"],["rs2276834"],["chr3"],["52291743"],[null],["A"],["G"],["NEUTRAL"],[null],["urinary_tract"],["carcinoma_--_NS"],["splice_acceptor_variant&amp;intron_variant"],["HIGH"],["transcript"],["Noncoding"],["n.626-2A&gt;G"],[""]],"container":"<table class=\"cell-border stripe\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>GENE<\/th>\n      <th>Number_SNVs_per_Gene<\/th>\n      <th>COSMIC_ID<\/th>\n      <th>rsID<\/th>\n      <th>CHROM<\/th>\n      <th>POS<\/th>\n      <th>ID<\/th>\n      <th>REF<\/th>\n      <th>ALT<\/th>\n      <th>FATHMM<\/th>\n      <th>SPLICEADJ<\/th>\n      <th>TISSUE<\/th>\n      <th>TUMOR<\/th>\n      <th>Annotation_1<\/th>\n      <th>Annotation_Impact_1<\/th>\n      <th>Feature_Type_1<\/th>\n      <th>Transcript_BioType_1<\/th>\n      <th>coding_DNA_change_1<\/th>\n      <th>protein_change_1<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[2,11]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Table 3. Non-Coding Variants, possibly pathogenic
</p>
</div>
<hr>
</div></div>

</div>
<div id="section-chromosome-and-gene-vizualization-of-mutations" class="section level1">
<div name="5_Chromosome_and_Gene_Vizualization_of_Mutations" data-unique="5_Chromosome_and_Gene_Vizualization_of_Mutations"></div><h1><span class="header-section-number">5</span> Chromosome and Gene Vizualization of Mutations</h1>
<p>Location Details coding falls within a coding region fiveUTR falls within a 5’ untranslated region threeUTR falls within a 3’ untranslated region intron falls within an intron region intergenic does not fall within a transcript associated with a gene spliceSite overlaps any portion of the first 2 or last 2 nucleotides of an intron promoter falls within a promoter region of a transcript</p>
<div class="sourceCode" id="section-cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb2-1"></a><span class="kw">kable</span>(<span class="kw">data.frame</span>())</span></code></pre></div>
<table>
<tbody>
<tr>
</tr>
</tbody>
</table>
<div class="figure" style="text-align: center">
<img src="./Expressed Variant Reporting_files/unnamed-chunk-16-1.png" alt="Figure 1. Percentage of different types of mutations identified." width="288">
<p class="caption">
Figure 1. Percentage of different types of mutations identified.
</p>
</div>
<div class="sourceCode" id="section-cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-1"></a>gene_tracks &lt;-<span class="st"> </span><span class="cf">function</span>(variants_df, vcf_s4, transriptsGR) {</span>
<span id="cb3-2"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-2"></a>  top5.coding &lt;-<span class="st"> </span>dplyr<span class="op">::</span><span class="kw">filter</span>(variants_df, </span>
<span id="cb3-3"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-3"></a>                     Transcript_BioType_<span class="dv">1</span> <span class="op">==</span><span class="st"> "Coding"</span>) <span class="op">%&gt;%</span><span class="st">  </span></span>
<span id="cb3-4"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-4"></a><span class="st">  </span>dplyr<span class="op">::</span><span class="kw">slice</span>(<span class="dv">1</span><span class="op">:</span><span class="dv">5</span>) <span class="op">%&gt;%</span><span class="st"> </span><span class="co">#Not accurate. Needs to be top5 unique genes. will deal with later. </span></span>
<span id="cb3-5"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-5"></a><span class="st">    </span><span class="kw">pull</span>(GENE)</span>
<span id="cb3-6"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-6"></a></span>
<span id="cb3-7"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-7"></a>  top5.coding &lt;-<span class="st"> </span>dplyr<span class="op">::</span><span class="kw">filter</span>(variants_df, Transcript_BioType_<span class="dv">1</span> <span class="op">==</span><span class="st"> "Coding"</span>) <span class="op">%&gt;%</span><span class="st"> </span></span>
<span id="cb3-8"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-8"></a><span class="st">    </span>dplyr<span class="op">::</span><span class="kw">filter</span>(GENE <span class="op">%in%</span><span class="st"> </span>top5.coding) <span class="op">%&gt;%</span><span class="st"> </span></span>
<span id="cb3-9"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-9"></a><span class="st">    </span><span class="kw">pull</span>(S4_Vector_IDs)</span>
<span id="cb3-10"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-10"></a></span>
<span id="cb3-11"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-11"></a>  </span>
<span id="cb3-12"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-12"></a>  vcf.ranges &lt;-<span class="st"> </span><span class="kw">rowRanges</span>(vcf_s4)</span>
<span id="cb3-13"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-13"></a>  vcf.ranges &lt;-<span class="st"> </span>vcf.ranges[top5.coding]</span>
<span id="cb3-14"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-14"></a>  vcf.ranges &lt;-<span class="st"> </span>vcf.ranges[<span class="kw">order</span>(vcf.ranges)]</span>
<span id="cb3-15"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-15"></a>  chrs &lt;-<span class="st"> </span><span class="kw">as.vector</span>(<span class="kw">seqnames</span>(vcf.ranges)<span class="op">@</span>values)</span>
<span id="cb3-16"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-16"></a>  chrs &lt;-<span class="st"> </span>chrs[<span class="kw">grep</span>(<span class="st">"chr1$"</span>, chrs)]</span>
<span id="cb3-17"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-17"></a>  <span class="kw">print</span>(chrs)</span>
<span id="cb3-18"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-18"></a>  </span>
<span id="cb3-19"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-19"></a>  <span class="cf">for</span> (chr <span class="cf">in</span> chrs){</span>
<span id="cb3-20"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-20"></a>      ranges.sub &lt;-<span class="st"> </span>vcf.ranges[<span class="kw">seqnames</span>(vcf.ranges)<span class="op">==</span>chr]</span>
<span id="cb3-21"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-21"></a>      transcripts.sub &lt;-<span class="st"> </span><span class="kw">suppressWarnings</span>(<span class="kw">subsetByOverlaps</span>(transriptsGR, ranges.sub))</span>
<span id="cb3-22"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-22"></a>      </span>
<span id="cb3-23"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-23"></a>      itrack<span class="fl">.1</span> &lt;-<span class="st"> </span><span class="kw">IdeogramTrack</span>(<span class="dt">genome =</span> <span class="st">"hg38"</span>, <span class="dt">chromosome =</span> <span class="st">"chr1"</span>)</span>
<span id="cb3-24"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-24"></a>      gtrack &lt;-<span class="st"> </span><span class="kw">GenomeAxisTrack</span>()</span>
<span id="cb3-25"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-25"></a>      grtrack &lt;-<span class="st"> </span><span class="kw">GeneRegionTrack</span>(transcripts.sub, </span>
<span id="cb3-26"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-26"></a>                                 <span class="dt">genome =</span> <span class="st">"hg38"</span>,</span>
<span id="cb3-27"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-27"></a>                                 <span class="dt">chromosome =</span> <span class="st">"chr1"</span>, </span>
<span id="cb3-28"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-28"></a>                                 <span class="dt">name =</span> <span class="st">"Gene Model"</span>,</span>
<span id="cb3-29"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-29"></a>                                  <span class="dt">showId=</span><span class="ot">TRUE</span>)</span>
<span id="cb3-30"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-30"></a>      track &lt;-<span class="st"> </span><span class="kw">AnnotationTrack</span>(ranges.sub,</span>
<span id="cb3-31"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-31"></a>                               <span class="dt">genome =</span> <span class="st">"hg38"</span>,</span>
<span id="cb3-32"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-32"></a>                               <span class="dt">name=</span><span class="st">"SNVs"</span>,</span>
<span id="cb3-33"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-33"></a>                               <span class="dt">ucscChromosomeNames=</span><span class="ot">FALSE</span>)</span>
<span id="cb3-34"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-34"></a>      </span>
<span id="cb3-35"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-35"></a>      <span class="kw">plotTracks</span>(<span class="kw">c</span>(itrack<span class="fl">.1</span>,gtrack,track,grtrack))</span>
<span id="cb3-36"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-36"></a>  }</span>
<span id="cb3-37"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-37"></a>  </span>
<span id="cb3-38"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb3-38"></a>}</span></code></pre></div>
<p>Error in .defaultRange(trackList, from = from, to = to, extend.left = extend.left, : Unable to automatically determine plotting ranges from the supplied track(s). Please provide range coordinates through the ‘from’ and ‘to’ arguments of the plotTracks function.</p>
<pre><code>## [1] "chr1"</code></pre>
<p><img src="./Expressed Variant Reporting_files/unnamed-chunk-19-1.png" width="864" style="display: block; margin: auto;"></p>
<hr>
</div>
<div id="section-expression-of-genes-with-mutations" class="section level1">
<div name="6_Expression_of_Genes_with_Mutations" data-unique="6_Expression_of_Genes_with_Mutations"></div><h1><span class="header-section-number">6</span> Expression of Genes with Mutations</h1>
<p>boxplots/violin plots</p>
<p>DE genes by condition</p>
<p>Sommelier results: Haplyotype/PCA</p>
<hr>
</div>
<div id="section-what-does-this-result-mean-for-you" class="section level1">
<div name="7_What_Does_This_Result_Mean_for_you" data-unique="7_What_Does_This_Result_Mean_for_you"></div><h1><span class="header-section-number">7</span> What Does This Result Mean for you?</h1>
<style type="text/css">
.watch-out {
  background-color: white;
  border: 3px solid red;
  font-weight: bold;
}
</style>
<div class="sourceCode" id="section-cb5"><pre class="sourceCode r watch-out"><code class="sourceCode r"><span id="cb5-1"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb5-1"></a><span class="co"># Need diagram here</span></span></code></pre></div>
<hr>
</div>
<div id="section-more-information" class="section level1">
<div name="8_More_information" data-unique="8_More_information"></div><h1><span class="header-section-number">8</span> More information</h1>
<div class="sourceCode" id="section-cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb6-1"></a><span class="co"># Add QR code where does it go?</span></span></code></pre></div>
<hr>
</div>
<div id="section-consent-for-analysis" class="section level1">
<div name="9_Consent_for_Analysis" data-unique="9_Consent_for_Analysis"></div><h1><span class="header-section-number">9</span> Consent for Analysis</h1>
<pre><code>## Please Sign Form. Acknowledment of informed consent is provided upon signing.</code></pre>
<div class="sourceCode" id="section-cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb8-1"></a><span class="kw">patient_signature</span>()</span></code></pre></div>
<iframe width="100%" height="400" class="shiny-frame" src="./Expressed Variant Reporting_files/saved_resource.html"></iframe>
<hr>
</div>
<div id="section-genome-references-and-software" class="section level1">
<div name="10_Genome_References_and_Software" data-unique="10_Genome_References_and_Software"></div><h1><span class="header-section-number">10</span> Genome References and Software</h1>
<style type="text/css">
.boxBorder {
     border: 2px solid #990066;
     padding: 10px;
     outline: #990066 solid 5px;
     outline-offset: 5px;
}
</style>
<div class="boxBorder">
<p>Additional information about the pipelines used, check out the github repositorites listed below. * <a href="https://github.com/collaborativebioinformatics/expressed-variant-impact">CTAT Mutation Pipeline for Input VCF</a> * <a href="https://github.com/collaborativebioinformatics/viravate2">Association between Genes and Drug Targets</a> * <a href="https://github.com/collaborativebioinformatics/mixed-sample-graphs">Mixed Sample Graphs QC</a> * <a href="https://github.com/collaborativebioinformatics/expressed-variant-reporting">snpReportR Generation</a></p>
</div>
<div id="htmlwidget-04fc60ff3b8b804ed0f2" style="width:100%;height:auto;" class="datatables html-widget html-widget-static-bound"><div id="DataTables_Table_3_wrapper" class="dataTables_wrapper no-footer"><div class="dataTables_length" id="DataTables_Table_3_length"><label>Show <select name="DataTables_Table_3_length" aria-controls="DataTables_Table_3" class=""><option value="10">10</option><option value="25">25</option><option value="50">50</option><option value="100">100</option></select> entries</label></div><div id="DataTables_Table_3_filter" class="dataTables_filter"><label>Search:<input type="search" class="" placeholder="" aria-controls="DataTables_Table_3"></label></div><table class="display dataTable no-footer" id="DataTables_Table_3" role="grid" aria-describedby="DataTables_Table_3_info">
  <thead>
    <tr role="row"><th class="sorting_disabled" rowspan="1" colspan="1" aria-label=" "> </th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_3" rowspan="1" colspan="1" aria-label="Value: activate to sort column ascending">Value</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_3" rowspan="1" colspan="1" aria-label="Value.1: activate to sort column ascending">Value.1</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_3" rowspan="1" colspan="1" aria-label="Value.2: activate to sort column ascending">Value.2</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_3" rowspan="1" colspan="1" aria-label="GATKCommandLine.CommandLine: activate to sort column ascending">GATKCommandLine.CommandLine</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_3" rowspan="1" colspan="1" aria-label="GATKCommandLine.Version: activate to sort column ascending">GATKCommandLine.Version</th><th class="sorting" tabindex="0" aria-controls="DataTables_Table_3" rowspan="1" colspan="1" aria-label="GATKCommandLine.Date: activate to sort column ascending">GATKCommandLine.Date</th></tr>
  </thead>
<tbody><tr role="row" class="odd"><td>SnpEffCmd</td><td>"SnpEff  -nostats -noLof -no-downstream -no-upstream hg38 "</td><td>"4.1k (build 2015-09-07), by Pablo Cingolani"</td><td>VCFv4.2</td><td>"HaplotypeCaller  --standard-min-confidence-threshold-for-calling 20.0 --recover-dangling-heads true --dont-use-soft-clipped-bases true --output /cromwell_root/OUTDIR/work_dir/chr1.interval.vcf --intervals /cromwell_root/OUTDIR/work_dir/chr1.intervals --input /cromwell_root/OUTDIR/work_dir/recalibrated.bam --reference ctat_genome_lib/ref_genome.fa  --use-new-qual-calculator true --annotate-with-num-discovered-alleles false --heterozygosity 0.001 --indel-heterozygosity 1.25E-4 --heterozygosity-stdev 0.01 --max-alternate-alleles 6 --max-genotype-count 1024 --sample-ploidy 2 --num-reference-samples-if-no-call 0 --contamination-fraction-to-filter 0.0 --output-mode EMIT_VARIANTS_ONLY --all-site-pls false --gvcf-gq-bands 1 --gvcf-gq-bands 2 --gvcf-gq-bands 3 --gvcf-gq-bands 4 --gvcf-gq-bands 5 --gvcf-gq-bands 6 --gvcf-gq-bands 7 --gvcf-gq-bands 8 --gvcf-gq-bands 9 --gvcf-gq-bands 10 --gvcf-gq-bands 11 --gvcf-gq-bands 12 --gvcf-gq-bands 13 --gvcf-gq-bands 14 --gvcf-gq-bands 15 --gvcf-gq-bands 16 --gvcf-gq-bands 17 --gvcf-gq-bands 18 --gvcf-gq-bands 19 --gvcf-gq-bands 20 --gvcf-gq-bands 21 --gvcf-gq-bands 22 --gvcf-gq-bands 23 --gvcf-gq-bands 24 --gvcf-gq-bands 25 --gvcf-gq-bands 26 --gvcf-gq-bands 27 --gvcf-gq-bands 28 --gvcf-gq-bands 29 --gvcf-gq-bands 30 --gvcf-gq-bands 31 --gvcf-gq-bands 32 --gvcf-gq-bands 33 --gvcf-gq-bands 34 --gvcf-gq-bands 35 --gvcf-gq-bands 36 --gvcf-gq-bands 37 --gvcf-gq-bands 38 --gvcf-gq-bands 39 --gvcf-gq-bands 40 --gvcf-gq-bands 41 --gvcf-gq-bands 42 --gvcf-gq-bands 43 --gvcf-gq-bands 44 --gvcf-gq-bands 45 --gvcf-gq-bands 46 --gvcf-gq-bands 47 --gvcf-gq-bands 48 --gvcf-gq-bands 49 --gvcf-gq-bands 50 --gvcf-gq-bands 51 --gvcf-gq-bands 52 --gvcf-gq-bands 53 --gvcf-gq-bands 54 --gvcf-gq-bands 55 --gvcf-gq-bands 56 --gvcf-gq-bands 57 --gvcf-gq-bands 58 --gvcf-gq-bands 59 --gvcf-gq-bands 60 --gvcf-gq-bands 70 --gvcf-gq-bands 80 --gvcf-gq-bands 90 --gvcf-gq-bands 99 --floor-blocks false --indel-size-to-eliminate-in-ref-model 10 --disable-optimizations false --just-determine-active-regions false --dont-genotype false --do-not-run-physical-phasing false --use-filtered-reads-for-annotations false --correct-overlapping-quality false --adaptive-pruning false --do-not-recover-dangling-branches false --dont-trim-active-regions false --max-extension 25 --padding-around-indels 150 --padding-around-snps 20 --kmer-size 10 --kmer-size 25 --dont-increase-kmer-sizes-for-cycles false --allow-non-unique-kmers-in-ref false --num-pruning-samples 1 --min-dangling-branch-length 4 --recover-all-dangling-branches false --max-num-haplotypes-in-population 128 --min-pruning 2 --adaptive-pruning-initial-error-rate 0.001 --pruning-lod-threshold 2.302585092994046 --max-unpruned-variants 100 --debug-assembly false --debug-graph-transformations false --capture-assembly-failure-bam false --error-correct-reads false --kmer-length-for-read-error-correction 25 --min-observations-for-kmer-to-be-solid 20 --likelihood-calculation-engine PairHMM --base-quality-score-threshold 18 --pair-hmm-gap-continuation-penalty 10 --pair-hmm-implementation FASTEST_AVAILABLE --pcr-indel-model CONSERVATIVE --phred-scaled-global-read-mismapping-rate 45 --native-pair-hmm-threads 4 --native-pair-hmm-use-double-precision false --bam-writer-type CALLED_HAPLOTYPES --min-base-quality-score 10 --smith-waterman JAVA --emit-ref-confidence NONE --max-mnp-distance 0 --force-call-filtered-alleles false --min-assembly-region-size 50 --max-assembly-region-size 300 --assembly-region-padding 100 --max-reads-per-alignment-start 50 --active-probability-threshold 0.002 --max-prob-propagation-distance 50 --force-active false --interval-set-rule UNION --interval-padding 0 --interval-exclusion-padding 0 --interval-merging-rule ALL --read-validation-stringency SILENT --seconds-between-progress-updates 10.0 --disable-sequence-dictionary-validation false --create-output-bam-index true --create-output-bam-md5 false --create-output-variant-index true --create-output-variant-md5 false --lenient false --add-output-sam-program-record true --add-output-vcf-command-line true --cloud-prefetch-buffer 40 --cloud-index-prefetch-buffer -1 --disable-bam-index-caching false --sites-only-vcf-output false --help false --version false --showHidden false --verbosity INFO --QUIET false --use-jdk-deflater false --use-jdk-inflater false --gcs-max-retries 20 --gcs-project-for-requester-pays  --disable-tool-default-read-filters false --minimum-mapping-quality 20 --disable-tool-default-annotations false --enable-all-annotations false --allow-old-rms-mapping-quality-annotation-data false"</td><td>"4.1.4.0"</td><td>"January 7, 2021 at 5:32:02 AM GMT"</td></tr></tbody></table><div class="dataTables_info" id="DataTables_Table_3_info" role="status" aria-live="polite">Showing 1 to 1 of 1 entries</div><div class="dataTables_paginate paging_simple_numbers" id="DataTables_Table_3_paginate"><a class="paginate_button previous disabled" aria-controls="DataTables_Table_3" data-dt-idx="0" tabindex="-1" id="DataTables_Table_3_previous">Previous</a><span><a class="paginate_button current" aria-controls="DataTables_Table_3" data-dt-idx="1" tabindex="0">1</a></span><a class="paginate_button next disabled" aria-controls="DataTables_Table_3" data-dt-idx="2" tabindex="-1" id="DataTables_Table_3_next">Next</a></div></div></div>
<script type="application/json" data-for="htmlwidget-04fc60ff3b8b804ed0f2">{"x":{"filter":"none","data":[["SnpEffCmd"],["\"SnpEff  -nostats -noLof -no-downstream -no-upstream hg38 \""],["\"4.1k (build 2015-09-07), by Pablo Cingolani\""],["VCFv4.2"],["\"HaplotypeCaller  --standard-min-confidence-threshold-for-calling 20.0 --recover-dangling-heads true --dont-use-soft-clipped-bases true --output /cromwell_root/OUTDIR/work_dir/chr1.interval.vcf --intervals /cromwell_root/OUTDIR/work_dir/chr1.intervals --input /cromwell_root/OUTDIR/work_dir/recalibrated.bam --reference ctat_genome_lib/ref_genome.fa  --use-new-qual-calculator true --annotate-with-num-discovered-alleles false --heterozygosity 0.001 --indel-heterozygosity 1.25E-4 --heterozygosity-stdev 0.01 --max-alternate-alleles 6 --max-genotype-count 1024 --sample-ploidy 2 --num-reference-samples-if-no-call 0 --contamination-fraction-to-filter 0.0 --output-mode EMIT_VARIANTS_ONLY --all-site-pls false --gvcf-gq-bands 1 --gvcf-gq-bands 2 --gvcf-gq-bands 3 --gvcf-gq-bands 4 --gvcf-gq-bands 5 --gvcf-gq-bands 6 --gvcf-gq-bands 7 --gvcf-gq-bands 8 --gvcf-gq-bands 9 --gvcf-gq-bands 10 --gvcf-gq-bands 11 --gvcf-gq-bands 12 --gvcf-gq-bands 13 --gvcf-gq-bands 14 --gvcf-gq-bands 15 --gvcf-gq-bands 16 --gvcf-gq-bands 17 --gvcf-gq-bands 18 --gvcf-gq-bands 19 --gvcf-gq-bands 20 --gvcf-gq-bands 21 --gvcf-gq-bands 22 --gvcf-gq-bands 23 --gvcf-gq-bands 24 --gvcf-gq-bands 25 --gvcf-gq-bands 26 --gvcf-gq-bands 27 --gvcf-gq-bands 28 --gvcf-gq-bands 29 --gvcf-gq-bands 30 --gvcf-gq-bands 31 --gvcf-gq-bands 32 --gvcf-gq-bands 33 --gvcf-gq-bands 34 --gvcf-gq-bands 35 --gvcf-gq-bands 36 --gvcf-gq-bands 37 --gvcf-gq-bands 38 --gvcf-gq-bands 39 --gvcf-gq-bands 40 --gvcf-gq-bands 41 --gvcf-gq-bands 42 --gvcf-gq-bands 43 --gvcf-gq-bands 44 --gvcf-gq-bands 45 --gvcf-gq-bands 46 --gvcf-gq-bands 47 --gvcf-gq-bands 48 --gvcf-gq-bands 49 --gvcf-gq-bands 50 --gvcf-gq-bands 51 --gvcf-gq-bands 52 --gvcf-gq-bands 53 --gvcf-gq-bands 54 --gvcf-gq-bands 55 --gvcf-gq-bands 56 --gvcf-gq-bands 57 --gvcf-gq-bands 58 --gvcf-gq-bands 59 --gvcf-gq-bands 60 --gvcf-gq-bands 70 --gvcf-gq-bands 80 --gvcf-gq-bands 90 --gvcf-gq-bands 99 --floor-blocks false --indel-size-to-eliminate-in-ref-model 10 --disable-optimizations false --just-determine-active-regions false --dont-genotype false --do-not-run-physical-phasing false --use-filtered-reads-for-annotations false --correct-overlapping-quality false --adaptive-pruning false --do-not-recover-dangling-branches false --dont-trim-active-regions false --max-extension 25 --padding-around-indels 150 --padding-around-snps 20 --kmer-size 10 --kmer-size 25 --dont-increase-kmer-sizes-for-cycles false --allow-non-unique-kmers-in-ref false --num-pruning-samples 1 --min-dangling-branch-length 4 --recover-all-dangling-branches false --max-num-haplotypes-in-population 128 --min-pruning 2 --adaptive-pruning-initial-error-rate 0.001 --pruning-lod-threshold 2.302585092994046 --max-unpruned-variants 100 --debug-assembly false --debug-graph-transformations false --capture-assembly-failure-bam false --error-correct-reads false --kmer-length-for-read-error-correction 25 --min-observations-for-kmer-to-be-solid 20 --likelihood-calculation-engine PairHMM --base-quality-score-threshold 18 --pair-hmm-gap-continuation-penalty 10 --pair-hmm-implementation FASTEST_AVAILABLE --pcr-indel-model CONSERVATIVE --phred-scaled-global-read-mismapping-rate 45 --native-pair-hmm-threads 4 --native-pair-hmm-use-double-precision false --bam-writer-type CALLED_HAPLOTYPES --min-base-quality-score 10 --smith-waterman JAVA --emit-ref-confidence NONE --max-mnp-distance 0 --force-call-filtered-alleles false --min-assembly-region-size 50 --max-assembly-region-size 300 --assembly-region-padding 100 --max-reads-per-alignment-start 50 --active-probability-threshold 0.002 --max-prob-propagation-distance 50 --force-active false --interval-set-rule UNION --interval-padding 0 --interval-exclusion-padding 0 --interval-merging-rule ALL --read-validation-stringency SILENT --seconds-between-progress-updates 10.0 --disable-sequence-dictionary-validation false --create-output-bam-index true --create-output-bam-md5 false --create-output-variant-index true --create-output-variant-md5 false --lenient false --add-output-sam-program-record true --add-output-vcf-command-line true --cloud-prefetch-buffer 40 --cloud-index-prefetch-buffer -1 --disable-bam-index-caching false --sites-only-vcf-output false --help false --version false --showHidden false --verbosity INFO --QUIET false --use-jdk-deflater false --use-jdk-inflater false --gcs-max-retries 20 --gcs-project-for-requester-pays  --disable-tool-default-read-filters false --minimum-mapping-quality 20 --disable-tool-default-annotations false --enable-all-annotations false --allow-old-rms-mapping-quality-annotation-data false\""],["\"4.1.4.0\""],["\"January 7, 2021 at 5:32:02 AM GMT\""]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Value<\/th>\n      <th>Value.1<\/th>\n      <th>Value.2<\/th>\n      <th>GATKCommandLine.CommandLine<\/th>\n      <th>GATKCommandLine.Version<\/th>\n      <th>GATKCommandLine.Date<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"order":[],"autoWidth":false,"orderClasses":false,"columnDefs":[{"orderable":false,"targets":0}]},"selection":{"mode":"multiple","selected":null,"target":"row","selectable":null}},"evals":[],"jsHooks":[]}</script>
<pre><code>## TxDb object:
## # Db type: TxDb
## # Supporting package: GenomicFeatures
## # Data source: /Users/work/expressed-variant-reporting/GenomicRefs/gencode.v22.chr_patch_hapl_scaff.annotation.gtf.gz
## # Organism: NA
## # Taxonomy ID: NA
## # miRBase build ID: NA
## # Genome: NA
## # Nb of transcripts: 213654
## # Db created by: GenomicFeatures package from Bioconductor
## # Creation time: 2021-01-07 19:02:13 -0800 (Thu, 07 Jan 2021)
## # GenomicFeatures version at creation time: 1.40.1
## # RSQLite version at creation time: 2.2.1
## # DBSCHEMAVERSION: 1.2</code></pre>
</div>
<div id="section-session-information" class="section level1">
<div name="11_Session_Information" data-unique="11_Session_Information"></div><h1><span class="header-section-number">11</span> Session Information</h1>
<div class="sourceCode" id="section-cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="http://127.0.0.1:7959/Report_v1_JSmith.Rmd#cb10-1"></a><span class="kw">sessionInfo</span>()</span></code></pre></div>
<pre><code>## R version 4.0.3 (2020-10-10)
## Platform: x86_64-apple-darwin17.0 (64-bit)
## Running under: macOS Mojave 10.14.6
## 
## Matrix products: default
## BLAS:   /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRblas.dylib
## LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib
## 
## locale:
## [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8
## 
## attached base packages:
##  [1] grid      stats4    parallel  stats     graphics  grDevices utils    
##  [8] datasets  methods   base     
## 
## other attached packages:
##  [1] GenomicFeatures_1.40.1      AnnotationDbi_1.50.3       
##  [3] Gviz_1.32.0                 VariantAnnotation_1.34.0   
##  [5] Rsamtools_2.4.0             Biostrings_2.56.0          
##  [7] XVector_0.28.0              SummarizedExperiment_1.18.2
##  [9] DelayedArray_0.14.1         matrixStats_0.57.0         
## [11] Biobase_2.48.0              GenomicRanges_1.40.0       
## [13] GenomeInfoDb_1.24.2         IRanges_2.22.2             
## [15] S4Vectors_0.26.1            BiocGenerics_0.34.0        
## [17] vcfR_1.12.0                 RColorBrewer_1.1-2         
## [19] gridExtra_2.3               ggplot2_3.3.2              
## [21] here_1.0.1                  stringr_1.4.0              
## [23] tidyr_1.1.2                 magrittr_1.5               
## [25] dplyr_1.0.2                 knitr_1.30                 
## [27] shiny_1.5.0                
## 
## loaded via a namespace (and not attached):
##   [1] backports_1.2.0          Hmisc_4.4-1              BiocFileCache_1.12.1    
##   [4] lazyeval_0.2.2           splines_4.0.3            BiocParallel_1.22.0     
##   [7] crosstalk_1.1.0.1        digest_0.6.27            ensembldb_2.12.1        
##  [10] htmltools_0.5.0          checkmate_2.0.0          memoise_1.1.0           
##  [13] BSgenome_1.56.0          cluster_2.1.0            askpass_1.1             
##  [16] prettyunits_1.1.1        jpeg_0.1-8.1             colorspace_1.4-1        
##  [19] ggrepel_0.8.2            blob_1.2.1               rvest_0.3.6             
##  [22] rappdirs_0.3.1           xfun_0.19                crayon_1.3.4            
##  [25] RCurl_1.98-1.2           jsonlite_1.7.1           survival_3.2-7          
##  [28] ape_5.4-1                glue_1.4.2               kableExtra_1.3.1        
##  [31] gtable_0.3.0             zlibbioc_1.34.0          webshot_0.5.2           
##  [34] scales_1.1.1             DBI_1.1.0                Rcpp_1.0.5              
##  [37] viridisLite_0.3.0        xtable_1.8-4             progress_1.2.2          
##  [40] htmlTable_2.1.0          foreign_0.8-80           bit_4.0.4               
##  [43] Formula_1.2-4            DT_0.17                  htmlwidgets_1.5.3       
##  [46] httr_1.4.2               ellipsis_0.3.1           farver_2.0.3            
##  [49] pkgconfig_2.0.3          XML_3.99-0.5             nnet_7.3-14             
##  [52] dbplyr_2.0.0             labeling_0.4.2           tidyselect_1.1.0        
##  [55] rlang_0.4.8              later_1.1.0.1            munsell_0.5.0           
##  [58] tools_4.0.3              generics_0.1.0           RSQLite_2.2.1           
##  [61] evaluate_0.14            fastmap_1.0.1            yaml_2.2.1              
##  [64] bit64_4.0.5              purrr_0.3.4              AnnotationFilter_1.12.0 
##  [67] nlme_3.1-150             mime_0.9                 formatR_1.7             
##  [70] xml2_1.3.2               biomaRt_2.44.4           compiler_4.0.3          
##  [73] rstudioapi_0.11          curl_4.3                 png_0.1-7               
##  [76] tibble_3.0.4             stringi_1.5.3            highr_0.8               
##  [79] memuse_4.1-0             lattice_0.20-41          ProtGenerics_1.20.0     
##  [82] Matrix_1.2-18            vegan_2.5-6              permute_0.9-5           
##  [85] vctrs_0.3.4              pillar_1.4.6             lifecycle_0.2.0         
##  [88] data.table_1.13.2        bitops_1.0-6             httpuv_1.5.4            
##  [91] rtracklayer_1.48.0       R6_2.5.0                 latticeExtra_0.6-29     
##  [94] promises_1.1.1           dichromat_2.0-0          MASS_7.3-53             
##  [97] assertthat_0.2.1         openssl_1.4.3            rprojroot_2.0.2         
## [100] withr_2.3.0              pinfsc50_1.2.0           GenomicAlignments_1.24.0
## [103] GenomeInfoDbData_1.2.3   mgcv_1.8-33              hms_0.5.3               
## [106] rpart_4.1-15             rmarkdown_2.5            biovizBase_1.36.0       
## [109] base64enc_0.1-3</code></pre>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("section-TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#section-TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "mathjax-local/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>


</div>
  </div>


</body></html>